src/test/java/org/opensearch/knn/indices/ModelDaoTests.java

```diff
@@ -107,6 +141,54 @@ public void testPut_withId() throws InterruptedException, IOException {
         assertTrue(inProgressLatch2.await(100, TimeUnit.SECONDS));
     }
 
+    public void testPut_withoutModel() throws InterruptedException, IOException {
```

Comment:
testPutWithoutModel

Reply:
I use underscore for better readability. First portion says what we are testing. After underscore represents specific case. I could just test all put together but I felt that would be too long.